lms/templates/learner_dashboard/course_card.underscore

```diff
@@ -1,13 +1,32 @@
-<img class="header-img" src="<%- image_url %>" alt="<%- display_name %>"/>
-<div class="course-details">
-    <h3 class="course-title"><%- display_name %></h3>
-    <p class="course-key"><%- key %></p>
-    <% if(enrollment_status){ %>
-        <span class="enrollment-info"><%- course_start %> - <%-course_end %></span>
-        <a href="<%- courseware_url %>" class="course-link cta"><%- gettext('View Course') %></a>
-    <% }else{ %>
-        <span class="enrollment-info"><%- gettext('Not Yet Enrolled') %></span>
-        <a href="<%- marketing_url %>" class="course-link cta"><%- gettext('Learn More') %></a>
-    <% } %>
+<div class="grid-container grid-manual">
+    <a href="<%- marketing_url %>" class="course-image-link col-12 xl-col-3 lg-col-3 md-col-3 sm-col-12">
```

Comment:
I added a comment about this in my last review. It doesn't make sense to apply these classes when the image is a fixed width. I had recommended a wrapper around the image and course title and text for the grid. What were your thoughts around that?